resources/assets/js/photoAlbum.js

```diff
@@ -0,0 +1,324 @@
+var items = [];
+var count = 0;
+var currentPhotoAlbum = photoAlbum.id;
```

Comment:
Als er geen photo album is dan geeft deze regel een error.

Reply:
fixed in commit 61ce0c6